ops/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Expose internal modules importing them in a controlled way."""
+
+from . import charm, framework, main, model, testing   # NOQA
```

Comment:
testing feels like one that I don't know we always want imported. It is the sort of module that I'd rather *not* be imported in production code. But it does break the pattern.

Reply:
I included it because we may be importing testing to use the Harness even not in pure unittests.